docs/design_guide.rst

```diff
@@ -383,6 +406,14 @@ Renders as:
 
   :param float degrees: Degrees to turn right
 
+Documentation References to other Libraries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+When you need to make references to documentation in other libraries you should refer the class using single
+backticks  ``:class:`~adafruit_motor.servo.Servo```. You must also add the reference in the ``conf.py`` file in the
+``intersphinx_mapping section`` adding a new entry::
```

Comment:
This should read `by adding a new entry`.

Reply:
Changed in https://github.com/adafruit/circuitpython/pull/4652/commits/cb42acea53d4780a018db3bea253708d5d7dd404